spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/SparkProcedures.java

```diff
@@ -37,6 +38,10 @@ public static ProcedureBuilder newBuilder(String name) {
     return builderSupplier != null ? builderSupplier.get() : null;
   }
 
+  public static Set<String> names() {
+    return BUILDERS.keySet();
```

Comment:
why not add the "system" here? do we want access to the list of names without the "system" prefix?

Reply:
to make it consistent with the existing method - the parameter name does not contain the `system.` prefix
```
public static ProcedureBuilder newBuilder(String name)
```